src/runtime/micro/device/arm/stm32f746xx/utvm_timer.c

```diff
@@ -0,0 +1,102 @@
+#ifdef __cplusplus
+extern "C" {
```

Comment:
.cc o be consistent with the rest part of the stack

Reply:
This is meant to be compiled and loaded on the device.  The `#ifdef __cplusplus` is just in case a C++ compiler is run over it.